portal/templates/redesign/about_new.html

```diff
@@ -0,0 +1,138 @@
+                    Chris Mukherjee, Chris Pavli, Christopher Brett, Cliford Bailey, Daniel Palma, Daniela Ortner, Dawid Chodura, Duncan Russell, Edd Armitage, Felix
+                    Stephenson, Florian Aucomte, Helen Mead, Isabel Richards, Jacq Train, James Dimmock, James Holland, James Smyth, Javi Carretero, Joe Slade,
+                    Joseph Zammit, Josh Blake, Katarzyna Zych, Kenny MacMillan, Kristian Brugts, Lloyd Allen, Lukasz Brzezowski, Maciej Zywiol, Mahana Clutha,
+                    Marcin Zieba, Maria Skrzypek, Mateusz Gizycki, Matt Whelan, Matthew Daggitt, Menelik Collymore, Michael Storey, Miguel Trigueira, Mike Bryant,
+                    Nel Sui, Niket Shah, Oliver Cartz, Oliver Jeeves, Patricia Botta, Paul Clarke, Paul Heideman, Paul Wright, Paulina Koch, Peter Hancock, Piotr
+                    &lsquo;Kru&rsquo; Kaczmar, Piotr Repetowski, Rachel Lord, Rafal Wawryk, Robert Brignull, Ruth Wall, Shubham Jain, Sophie Whelan, Steph Lovis,
+                    Stephen Kennedy, Thomas Cassany, Tricy Dino, Varadh Kalidasan, Wael Al Jeshi, Will Peck, Yongji Zhou.</i></p>
+            </div>
+            <div class="col-sm-6">
+                <h3>Want to get involved?</h3><br>
+                <p class="body-text">We are open source, so anyone can ask to contribute. You can play with game-running JavaScript, Python/Django, animation
+                    using SGV and Raphael, and a lot more. We&rsquo;d like input from all sorts of backgrounds, whether you&rsquo;re: a programmer looking for a
+                    creative outlet; a teacher hoping to shape the resources;
+                    <a target="_blank" href="https://crowdin.com/project/code-for-life/invite/public?show_welcome&d=876e8507d82d9e07a09b4696a88c0c02479434">
+                        a polyglot who&rsquo;d like to help with translation</a>;
+                    or even a pupil putting your skills to the test.</p><br><br><br>
+                <h4>Developers</h4><br>
+                <p class="body-text">To contribute, head over to <a target="_blank" href="https://github.com/ocadotechnology/codeforlife-portal">GitHub</a>,
+                    check out the issue tracker, and get started. There you can suggest new features or assign yourself an issue to develop (you&rsquo;ll find
+                    more info about how to do this on GitHub).</p><br><br><br>
+                <h4>Teachers, parents, and creatives</h4><br>
+                <p class="body-text">Please get in touch through our <a href="{% url 'help_new' %}">contact</a> form and let us know how you would like to get involved.</p>
+            </div>
+        </div>
+    </div>
+    <section class="col-sm-7 col-center">
+        <h3>We would like to thank our friends who <br>have contributed to this initiative</h3>
+        <div class="row row--regular image-row">
+            <img src='{% static "portal/img_new/10x_logo.png" %}'>
```

Comment:
Alt and Title attributes have been added to each image, except the ones which are to be removed soon (ie the arrow images in the onboarding).